README.md

```diff
@@ -11,7 +11,20 @@ Checkup was created by Matt Holt, author of the [Caddy web server](https://caddy
 
 This tool is a work-in-progress. Please use liberally (with discretion) and report any bugs!
 
+## Recent changes
 
+Due to recent development, some breaking changes have been introduced:
+
+- providers: the json config field `provider` was renamed to `type` for consistency,
+- notifiers: the json config field `name` was renamed to `type` for consistency,
+- sql: by default the sqlite storage engine is disabled (needs build with `-tags sql` to enable),
```

Comment:
If there's any way to call this out more aggressively, that would be great! Perhaps a CHANGELOG? My crons stopped running overnight due to (1) me using r.j3ss.co/checkup:latest, which tracks HEAD and (2) these changes. Specifically, the `provider` -> `type` change for `storage` caught me. For anyone relying on HEAD for their production checks, this is likely to bite them.

Reply:
I'm very sorry about that. It was a risk I was aware of when I was cleaning up the code for this, and I just hoped people aren't blindly doing nightly/HEAD builds, and tried to make it as obvious as possible that there was a breaking change introduced by prominently updating the README at the very top.

Aside from adding a CHANGELOG file,

1. we can decide on a branching model (master|develop, v0, v1...), AND
2. we can remove tags as they are unused/out of date (#113 #86 #84 ...) OR
3. we can tag a 0.3.0 before breaking changes and 0.4.0 after

And finally, we could automate the release process in line with what we choose above (#96). There are also a number of issues that could be solved by having a rolling release model:

- #113 
- #86
- #84 (this is an issue that just needs a new tagged build)
- #46 

The current releases are woefully out of date. I'm voting to remove them, keep the development on master, branch to v0 before the breaking changes ([f4747239](https://github.com/sourcegraph/checkup/commit/f47472392057b3cc432fcd426e61f0a56d64c697)), and branch into v1 later when we feel the release is pretty stable and non-breaking.

@beyang thoughts?

Reply:
@parkr sorry this bit you. We don't have a good sense of who's using this at the moment, so yes, we should probably adopt a versioning system as @titpetric proposes. @titpetric the model you propose sounds good to me. I'd propose this model:
* `master` is the development branch
* We adopt semver. I've pushed tags `v1.0.0` (the last commit before @titpetric's recent changes) and `v2.0.0` (the latest `master` commit). I've also pushed a `1.0` branch in case we want to cut any new patches off the older version

@titpetric feel free to push up a `CHANGELOG` file as well. Also not we did publish release notes for early versions of Checkup: https://github.com/sourcegraph/checkup/releases.